src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/managedexports.cpp

```diff
@@ -601,4 +601,14 @@ http_reset_stream(
     pHttpResponse->ResetStream(errorCode);
 }
 
+EXTERN_C __MIDL_DECLSPEC_DLLEXPORT
+HRESULT
+http_response_set_need_goaway(
+    _In_ IN_PROCESS_HANDLER* pInProcessHandler
+    )
+{
+    IHttpResponse4* pHttpResponse = (IHttpResponse4*)pInProcessHandler->QueryHttpContext()->GetResponse();
+    pHttpResponse->SetNeedGoAway();
```

Comment:
Need -> Needs?

Reply:
That's the IIS api name, I don't think it's worth changing at this point.